packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFWebsiteDataStoreHostApiTests.m

```diff
@@ -70,6 +70,8 @@ - (void)testRemoveDataOfTypes {
                                         blockError = error;
                                       }];
   XCTAssertEqualObjects(returnValue, @YES);
+  // Asserts whether the NSNumber value represents a BOOL.
```

Comment:
Nit: I would say "will be deserialized by the standard codec as a boolean" since that's the thing that matters here. "Represents" is ambiguous since even without the change `boolValue` can be called on it and will give the expected value, so arguably it represents a bool regardless.